cloud/services/virtualmachines/virtualmachines.go

```diff
@@ -434,7 +434,9 @@ func getResourceNameByID(resourceID string) string {
 	return resourceName
 }
 
-func getSpotVMOptions(spotVMOptions *infrav1.SpotVMOptions) (compute.VirtualMachinePriorityTypes, compute.VirtualMachineEvictionPolicyTypes, *compute.BillingProfile, error) {
+// GetSpotVMOptions takes the spot vm options
+// and returns the indivial vm priority, eviction policy and billing profile
+func GetSpotVMOptions(spotVMOptions *infrav1.SpotVMOptions) (compute.VirtualMachinePriorityTypes, compute.VirtualMachineEvictionPolicyTypes, *compute.BillingProfile, error) {
```

Comment:
let's move this helper out of virtualmachines since it's used by vmss too now. An appropriate place for it would be in a new `spotinstances.go` file under `cloud/converters`

Reply:
Done